README.md

```diff
@@ -6,5 +6,17 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+Rails is a framework for building websites. As such, Rails establishes conventions for easier collaboration and maintenance. These conventions are codified as the Rails API
+
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
+Picture added
+
 1. Describe the flow of a PATCH request as it goes through the app.
+The PATCH method requests that a set of changes described in the request entity be applied to the resource identified by the Request- URI. The set of changes is represented in a format called a "patch document" identified by a media type. If the Request-URI does not point to an existing resource, the server MAY create a new resource, depending on the patch document type and permissions, etc.
+
+With PATCH, the enclosed entity contains a set of instructions describing how a resource currently residing on the origin server should be modified to produce a new version.
+
+The PATCH method affects the resource identified by the Request-URI, and it also MAY have side effects on other resources; i.e., new resources may be created, or existing ones modified, by the application of a PATCH.
```

Comment:
OK... but how does it make its way through Rails? What happens when a Rails application receives a PATCH request?